src/router.js

```diff
@@ -13,8 +13,8 @@ export const router = {
       // Health-check route. Used by platform to check if service is running, do not remove!
       await server.register([health])
 
-      // Application specific routes
-      await server.register([cases])
+      // Domain specific routes
+      await server.register([caseManagement])
```

Comment:
Let's align plugin structure with CW backend and GAS. Remove router and register plugins direct in `server.js`